gen/gen.go

```diff
@@ -0,0 +1,26 @@
+package main
+
+import (
+	multisig "github.com/filecoin-project/specs-actors/actors/builtin/multisig"
+	gen "github.com/whyrusleeping/cbor-gen"
+)
+
+func main() {
+
+	// General Types
+	if err := gen.WriteTupleEncodersToFile("./actors/builtin/multisig/cbor_gen.go", "multisig",
+		// actor state
+		multisig.MultiSigActorState{},
+		multisig.MultiSigTransaction{},
+		// method params
+		multisig.ConstructorParams{},
+		multisig.ProposeParams{},
+		multisig.TxnIDParams{},
+		multisig.ChangeNumApprovalsThresholdParams{},
+		multisig.SwapAuthorizedPartyParams{},
+		// method returns
+		multisig.ProposeReturn{},
```

Comment:
@hannahhoward you somewhere had a handy-looking Go annotation that put this information in the code itself. Is that something we could use here?

Reply:
I think this is it - github.com/hannahhoward/cbor-gen-for. I was unable to get it working, this might be due to some env vars being required: https://github.com/hannahhoward/cbor-gen-for/blob/master/main.go#L15.